roles/vm/tasks/main.yml

```diff
@@ -0,0 +1,40 @@
+---
+# VM Creation testings.
+# Missing: reclaim option on disk, HA settings
+
+# TODO
+# - Add the VM to the target HA Group
+
+- name: Create VM
+  community.general.proxmox_kvm:
+    api_user: '{{ proxmox_api_user }}'
+    api_token_id: '{{ proxmox_api_token_id }}'
+    api_token_secret: '{{ proxmox_api_token_secret }}'
+    api_host: '{{ proxmox_api_host }}'
+    node: '{{ item.node }}'
+    name: '{{ item.name }}'
+    vmid: '{{ item.vmid }}'
+    net: '{{ item.net }}'
+    cores: '{{ item.cores }}'
+    memory: '{{ item.memory }}'
+    virtio: '{{ item.virtio }}'
+  retries: 10
+  delay: 3
+  register: vm
+  until: vm.vmid is defined
+  loop: '{{ pve_kvm }}'
```

Comment:
Can you name it `proxmox_vm` instead since the role is `mila.proxmox.vm`. See https://github.com/mila-iqia/ansible-collection-proxmox/blob/main/roles/container/tasks/create.yml#L33 for `mila.proxmox.container`.

Reply:
yep, done